### PR TITLE
Update README.md for user documentation

### PR DESCRIPTION
**Pull request by junhu666**
Update README.md, need feedback to add-in 
**How to report a bug**
and 
**Known bugs**
section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,9 +81,17 @@ Executing the systems of this service require different steps:
 > Press the green button next to the `main` method in `Session.java` and the server will be executed:
 ![executionImage](readmeresources/backendexecution.png)
 
+
+
+## How to report a bug
+No sure what to put on this section for now
+
+##  Known bugs
+- N/A
```
Comment (Esauidk) [inline README.md:90]: Put a hyperlink to here https://github.com/ngutn24/RollBuddy/issues?q=is%3Aopen+is%3Aissue+label%3Abug

File: README.md [hunk 2]
```diff
@@ -65,9 +81,17 @@ Executing the systems of this service require different steps:
 > Press the green button next to the `main` method in `Session.java` and the server will be executed:
 ![executionImage](readmeresources/backendexecution.png)
 
+
+
+## How to report a bug
+No sure what to put on this section for now
```
Comment (Esauidk) [inline README.md:87]: Direct them to making github issues for bug submissions, tell them to the the template

Comment (Esauidk) [reply README.md:87]: Also say to add the `bug` label before submitting the issue

**Review by Esauidk — APPROVED** [review]

**Review by junhu666 — COMMENTED** [review]
Looks good

**Pull request merged**